mne/time_frequency/multitaper.py

```diff
@@ -63,7 +63,12 @@ def dpss_windows(N, half_nbw, Kmax, *, sym=True, norm=None, low_bias=True):
     ----------
     .. footbibliography::
     """
-    dpss, eigvals = sp_dpss(N, half_nbw, Kmax, sym=sym, norm=norm, return_ratios=True)
+    if N <= 1:  # workaround for https://github.com/scipy/scipy/pull/22344
```

Comment:
this comment should have "TODO remove when SciPy min version is 1.16" or similar, right? (I forget what exact wording we agreed upon --- I think it was to include both words "todo" and "version" somewhere)